frontend/src/app/modules/interventions/map/map-filter/map-filter.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, OnInit, EventEmitter, Output } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+
+import { ComponentWithSubscriptions } from '@shared/components/base';
+import { InterventionStatus, InterventionStatusOptions } from '@shared/domain/intervention.status';
+
+export interface InterventionFilters {
+  Status: InterventionStatus;
```

Comment:
Not really a mistake, but I am not really used to have properties starting with uppercase letter. This seems like a bleedthrough from C# code for me :P

How do you feel about it personally?

As far as I know, Azure Functions will handle the param properly if it starts with lowercase anyway. Also, I see that you are typecasting it anyway before sending, so we might switch it to uppercase just before sending if necessary ;)

For JS cleanliness I would keep it as lowercase, but please use your own judgement here. Just wanted to let you know :)

Reply:
To be honest I don't have problem with that, I wouldn't complicate the code just for the matter of personal preference.
For a long time I was using uppercase CSS BEM class names according, e.g. `.AppComponent-text` and I've really liked it and even if it's not a common practice I think in the end it was easier to read such BEM code.